### PR TITLE
#428 Removing Unused properties in Server config

### PR DESCRIPTION
**Pull request by manishsahni2000**

### CODE REVIEW

**Comment by manishsahni2000** [comment]
Hi,

I have incorporated the above-mentioned comments.
Please have a look 

Thanks

File: components/proxy/src/main/java/com/hotels/styx/ServerConfigSchema.java [hunk 1]
```diff
@@ -73,12 +73,17 @@
                                     optional("bossThreadsCount", integer()),
                                     optional("clientWorkerThreadsCount", integer()),
                                     optional("workerThreadsCount", integer()),
+                                    // tcpNoDelay is deprecated by PR #464
                                     optional("tcpNoDelay", bool()),
+                                    // nioReuseAddress is deprecated by PR #464
                                     optional("nioReuseAddress", bool()),
+                                    // nioKeepAlive is deprecated by PR #464
                                     optional("nioKeepAlive", bool()),
+                                    // maxInitialLength is deprecated by PR #464
```
Comment (mikkokar) [inline components/proxy/src/main/java/com/hotels/styx/ServerConfigSchema.java]: `maxInitialLength` is still used. Please remove the comment.

File: components/proxy/src/main/java/com/hotels/styx/ServerConfigSchema.java [hunk 2]
```diff
@@ -88,12 +93,17 @@
                                     field("connectors", serverConnectorsSchema),
                                     optional("bossThreadsCount", integer()),
                                     optional("workerThreadsCount", integer()),
+                                    // tcpNoDelay is deprecated by PR #464
                                     optional("tcpNoDelay", bool()),
+                                    // nioReuseAddress is deprecated by PR #464
                                     optional("nioReuseAddress", bool()),
+                                    // nioKeepAlive is deprecated by PR #464
                                     optional("nioKeepAlive", bool()),
+                                    // maxInitialLength is deprecated by PR #464
```
Comment (mikkokar) [inline components/proxy/src/main/java/com/hotels/styx/ServerConfigSchema.java]: Also here. Remove the comment. `maxInitialLength` is still relevant.

File: components/proxy/src/main/java/com/hotels/styx/ServerConfigSchema.java (hunk 1 above)
Comment (manishsahni2000) [reply components/proxy/src/main/java/com/hotels/styx/ServerConfigSchema.java]: Done

File: components/proxy/src/main/java/com/hotels/styx/ServerConfigSchema.java (hunk 2 above)
Comment (manishsahni2000) [reply components/proxy/src/main/java/com/hotels/styx/ServerConfigSchema.java]: Done

**Comment by manishsahni2000** [comment]
Incorporated the changes

**Pull request merged**